packages/io-services-cms-models/openapi/services-cms-schemas.yaml

```diff
@@ -0,0 +1,251 @@
+    metadata:
+      $ref: "#/ServiceMetadata"
+  required:
+    - name
+    - description
+    - organization
+    - metadata
+ServiceMetadata:
+  type: object
+  description: A set of service metadata properties
+  properties:
+    web_url:
+      type: string
+      minLength: 1
+    app_ios:
+      type: string
+      minLength: 1
+    app_android:
+      type: string
+      minLength: 1
+    tos_url:
+      type: string
+      minLength: 1
+    privacy_url:
+      type: string
+      minLength: 1
+    address:
+      type: string
+      minLength: 1
+    phone:
+      type: string
+      minLength: 1
+    email:
+      type: string
+      minLength: 1
+    pec:
+      type: string
+      minLength: 1
+    cta:
+      type: string
+      minLength: 1
+    token_name:
+      type: string
+      minLength: 1
+    support_url:
+      type: string
+      minLength: 1
+    category:
+      type: string
+      enum: [SPECIAL, STANDARD]
+    custom_special_flow:
+      type: string
+      minLength: 1
+    scope:
+      type: string
+      enum: [NATIONAL, LOCAL]
+  required:
+    - scope
+ServiceLifecycleStatus:
+  type: object
+  properties:
+    value:
+      $ref: "#/ServiceLifecycleStatusType"
+    reason:
+      description: Reason for status value
+      type: string
+  required:
+    - value
+ServiceLifecycleStatusType:
+  description: |
+    Service lifecycle status<br>
+    - _draft_: A new draft of the service has been created, filled in completely or partially and saved by the system.
+    - _submitted_: The draft has been sent for internal validation to PagoPA and the response is awaited. In this state, the service is
+    immutable and is frozen in the shipped version.
+    - _approved_: The service has been approved by the _PagoPA S.p.A._ internal validation process, it is correct and suitable for publication.
+    - _rejected_: The service goes back to the draft but cannot be resubmitted for validation unless it is modified by the institution in
+    at least one of its fields.
+    - _deleted_: The service is permanently deleted from the Back-Office.
+  type: string
+  enum: [draft, submitted, approved, rejected, deleted]
+ServicePublicationStatusType:
+  description: |
+    Service publication status<br>
+    - _published_: The service is visible in _IO App_ to users
+    - _unpublished_: The service is no longer visible in _IO App_ to users, but remains in the institution's Back-Office, which can
+    later choose to publish it again.
+  type: string
+  enum: [published, unpublished]
+ServicePagination:
+  type: object
+  properties:
+    value:
+      type: array
+      items:
+        $ref: "#/ServiceLifecycle"
+    pagination:
+      $ref: "#/PaginationResultSet"
+PaginationResultSet:
+  type: object
+  properties:
+    offset:
+      type: number
+      description: result set offset
+    limit:
+      type: number
+      description: result set size
+    count:
+      type: number
+      description: total record count
```

Comment:
@rodoviti unfortunately I can't do that because `PaginationResultSet` is used inside the file by `ServicePagination`.
I believe these types of definition files must be "self-consistent" and not have external references.